connector/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -333,6 +333,9 @@ message PythonUDF {
   bytes command = 3;
   // (Required) Python version being used in the client.
   string python_ver = 4;
+  // (Optional) Whether this PythonUDF should be executed in barrier mode.
```

Comment:
Can you briefly describe in the comment what barrier mode is used for?

Reply:
good point, will update. thx